src/ffi.rs

```diff
@@ -272,12 +272,16 @@ extern "C" {
 //  In:      flags: which parts of the context to initialize.
 pub unsafe extern "C" fn secp256k1_context_create(flags: c_uint) -> *mut Context {
     assert!(mem::align_of::<usize>() >= mem::align_of::<u8>());
+    assert_eq!(mem::size_of::<usize>(), mem::size_of::<&usize>());
 
-    let size: usize =  secp256k1_context_preallocated_size(flags)/mem::size_of::<usize>();
-
+    let mut size: usize =  secp256k1_context_preallocated_size(flags);
+    {
+        let s = mem::size_of::<usize>();
+        size = size/s + if size % s != 0 { 1 } else { 0 };
+    }
```

Comment:
I think it would be more idiomatic to write
```
let word_size = mem::size_of::<usize>();
let n_words = (secp256k1_context_preallocated_size(flags) + word_size - 1) / word_size;
let buf = vec![0usize, n_words].into_boxed_slice();
```

Reply:
Done